docs/Tutorial_Consensus_Motif.ipynb

```diff
@@ -0,0 +1,610 @@
+{
```

Comment:
<p>Great example!</p><p> </p><p>Please add <code>plt.show()</code>  to the end of the cell to prevent <code>Text(0.5, 0.98, 'Vertical eye position while spelling different sentences')</code> from being displayed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>